components/Accordion/index.d.ts

```diff
@@ -2,8 +2,8 @@ import React from 'react';
 
 export interface AccordionProps {
     theme?: {
-        colors?: object;
-        spacing?: object;
+        colors?: {};
+        spacing?: {};
```

Comment:
You could use `Record<>`  instead of `{}` - https://www.typescriptlang.org/docs/handbook/utility-types.html#recordkeys-type

```suggestion
        colors?: Record<string, unknown>;
        spacing?: Record<string, unknown>;
```

Reply:
Man, this is interesting. I would like to create an interface to  theme items and include it on ts definitions, but we're considering set a ThemeProvider and remove theme from all components. I don't know if worth the effort.

Reply:
No problem! If you'll use ThemeProvider, this is deprecated soon. 😎